src/app/zap-templates/zcl/data-model/chip/color-control-cluster.xml

```diff
@@ -17,48 +17,35 @@ limitations under the License.
 <configurator>
 
 
-  <enum name="HueDirection" type="enum8">
+  <enum name="DirectionEnum" type="enum8">
     <cluster code="0x0300"/>
-    <item name="ShortestDistance" value="0x0"/>
-    <item name="LongestDistance" value="0x1"/>
+    <item name="Shortest" value="0x0"/>
+    <item name="Longest" value="0x1"/>
     <item name="Up" value="0x2"/>
     <item name="Down" value="0x3"/>
   </enum>
 
-  <enum name="HueMoveMode" type="enum8">
+  <enum name="MoveModeEnum" type="enum8">
     <cluster code="0x0300"/>
     <item name="Stop" value="0x0"/>
     <item name="Up" value="0x1"/>
     <item name="Down" value="0x3"/>
   </enum>
 
-  <enum name="HueStepMode" type="enum8">
+  <enum name="StepModeEnum" type="enum8">
     <cluster code="0x0300"/>
     <item name="Up" value="0x1"/>
     <item name="Down" value="0x3"/>
   </enum>
 
-  <enum name="SaturationMoveMode" type="enum8">
-    <cluster code="0x0300"/>
-    <item name="Stop" value="0x0"/>
-    <item name="Up" value="0x1"/>
-    <item name="Down" value="0x3"/>
-  </enum>
-
-  <enum name="SaturationStepMode" type="enum8">
-    <cluster code="0x0300"/>
-    <item name="Up" value="0x1"/>
-    <item name="Down" value="0x3"/>
-  </enum>
-
-  <enum name="ColorMode" type="enum8">
+  <enum name="ColorModeEnum" type="enum8">
     <cluster code="0x0300"/>
     <item name="CurrentHueAndCurrentSaturation" value="0x00"/>
     <item name="CurrentXAndCurrentY" value="0x01"/>
     <item name="ColorTemperature" value="0x02"/>
   </enum>
 
-  <bitmap name="ColorCapabilities" type="bitmap16">
+  <bitmap name="ColorCapabilitiesBitmap" type="bitmap16">
     <cluster code="0x0300"/>
     <field name="HueSaturationSupported" mask="0x0001"/>
```

Comment:
all *Supported have been removed from the names
HueSaturation
EnhancedHue
ColorLoop
XY
ColorTemperature